### PR TITLE
Erinhales/clean env

### LINKED ISSUE #291: bug: minutes remaining flickers
When the confirmAt check fails, we fall back on calculating from relayedAt and add a 10 minute buffer.  This is causing minutes remaining to flicker +- 10 minutes

### PR DESCRIPTION
**Pull request by ErinHales**
Closes #291 #303

- Places environment-specific variables in env files
- Uses optimisticSeconds instead of converting to minutes
- Fine-tunes status/confirmAt logic (don't do expensive queries if confirmAt is already defined)
- Fixes header responsiveness (shouldn't need to keep setting confirmAt in parent component to update child)

### CODE REVIEW

File: src/views/Transaction/Nomad/Header.vue [hunk 1]
```diff
@@ -221,6 +226,13 @@ export default defineComponent({
       notification,
     }
   },
+
+  mounted() {
+    this.now = Date.now()
+    setTimeout(() => {
```
Comment (Imti) [inline src/views/Transaction/Nomad/Header.vue:232]: Should this be a setInterval?

File: src/views/Transaction/Nomad/Header.vue [hunk 2]
```diff
@@ -209,9 +214,9 @@ export default defineComponent({
     AlertCircleOutline,
   },
   data: () => ({
-    PROCESS_TIME_IN_MINUTES,
     showStatus: false,
-  }),
+    now: undefined
```
Comment (Imti) [inline src/views/Transaction/Nomad/Header.vue:218]: We could set Date.now() here, unless we specifically want to wait until after component mount to set it

**Review by Imti — APPROVED** [review]
Some minor comments 🐙

File: src/views/Transaction/Nomad/Header.vue (hunk 2 above)
Comment (ErinHales) [reply src/views/Transaction/Nomad/Header.vue:218]: It wasn't triggering the update... But then I refactored a bit, so maybe I'll try again

**Pull request merged**